build/podman/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>podman-build</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: Podman-build</name>
+    <properties>
+        <quarkus.container-image.build>true</quarkus.container-image.build>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-container-image-podman</artifactId>
+        </dependency>
+    </dependencies>
+    <profiles>
+        <!-- Skipped on Windows when availability of Linux containers is not explicitly declared -->
```

Comment:
I am thinking - this `skip-tests-on-windows` will also skip tests when Podman/Docker is available, unless we set property. We could get rid of this if we skipped Quarkus JAR/native build with Quarkus maven plugin with `<quarkus.build.skip>true</quarkus.build.skip>` and let framework to perform the build only when Linux containers are available.

Reply:
full disclosure: we use Mave Quarkus plugin when it is a native build to build the executable and we could adjust FW to do that as well, but OK. Closing this thread.